1906-11-26.xml

```diff
@@ -0,0 +1,1934 @@
+                    <div type="advert" xml:id="deg-ad-ccc01">
+                        <head>The Cigarettes Manufactured by<lb/>The Cleopatra Cigarette Co.</head>
+                        <p>G. NUNGOYICH</p>
+                        <p>are on sale at the Company's establishment by Grand Contental Hotel,
+                            Cairo, and at Walker &amp; Meimarschi's, Alexandria.</p>
+                        <p>Purveyors to H. H. the KHEDIVE.</p>
+                        <p> Patronized by the Duke of Connaught and the Archduke Otto and all the
+                            High Life of Egypt.<measure type="indexNo">35750 18-4-80</measure></p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-aan03">
+                        <head>Hamburg &amp; Anglo-American Nile Co.</head>
+                        <p>Weekly departure during Winter Season by the<lb/> Luxurious First Class
+                            Tourist Steamers VICTORIA, PURITAN &amp; MAYFLOWER.<lb/> Regular weekly
+                            Departures to the SECOND CATARACT by the S.S. INDIANA.<lb/> THROUGH
+                            BOOKINGS TO KHARTOUM, GONDOKORO AND THE WHITE NILE.<lb/> Steamers and
+                            Dahabeahs for private charter. Steam Tugs and Steam Launches for
+                            hire.<lb/> FREIGHT SERVICE BY STEAM BARGES BETWEEN CAIRO AND
+                            ALEXANDRIA.<lb/> Working in conjunction and under special arrangement
+                            with the<lb/> “Upper Egypt Hotels Company."</p>
+                        <p>For details and illustrated programmes apply to "THE HAMBURG and
+                            ANGLO-AMERICAN<lb/> NILE COMPANY."</p>
+                        <p> OFFICES IN CAIRO: Sharia Boulac, "Grand Continental Hotel Buildings.”
+                                <measure type="indexNo">31-3-07</measure></p>
+                        <p>Hamburg-Amerika Linie.</p>
+                        <p>Return to Egypt<lb/> by the new S.S. Oceana of 9000 Tons, 350 berths
+                            including 90 single cabins, greatest comfort, best cuisine.</p>
+                        <p>FARES: Genoa to Alexandria from £14. Naples to Alexandria from £10.</p>
+                        <table>
+                            <row>
+                                <cell>Leave Alexandria</cell>
+                                <cell><date when="1906-10-23">8 November</date> p.m.</cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1906-10-27">13 November</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-06">10 &amp; 24 November</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-10">27 November
```

Comment:
Here and afterwards, the problem is that your text is not wrapped in `<div>` tags and in `<p>` tags. All text in xml must be wrapped in one or both of these tags. I can help you with this in class or in office hours.